trp_use_case/view/use_case_collection.xml

```diff
@@ -14,8 +14,18 @@
                     </header>
                     <sheet>
                         <group>
+                            <div class="oe_title">
+                                <label for="name" string="Name" class="oe_edit_only"/>
+                                <h1><field name="name"/></h1>
+                            </div>
+                            <div name="buttons" class="oe_right oe_button_box">
+                                <button class="oe_stat_button" type="action" icon="fa-tasks"
+                                    name="%(trp_use_case.action_use_case_collection_open_use_cases)d"
+                                    string="" >
```

Comment:
you don't need to set this to an empty string, just don't pass the attribute at all